instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/logs/LoggerE2ETests.kt

```diff
@@ -250,4 +270,246 @@ class LoggerE2ETests {
             )
         }
     }
+
+    /**
+     * apiMethodSignature: Logger#fun addAttribute(String, Boolean)
+     */
+    @Test
+    fun logs_logger_add_boolean_attribute() {
+        val testMethodName = "logs_logger_add_boolean_attribute"
+        measure(testMethodName) {
+            logger.addAttribute(SPECIAL_BOOL_ATTRIBUTE_NAME, forge.aBool())
+        }
+        logger.sendRandomLog(testMethodName, forge)
+    }
+
+    /**
+     * apiMethodSignature: Logger#fun addAttribute(String, Int)
+     */
+    @Test
+    fun logs_logger_add_int_attribute() {
+        val testMethodName = "logs_logger_add_int_attribute"
+        measure(testMethodName) {
+            logger.addAttribute(SPECIAL_INT_ATTRIBUTE_NAME, CUSTOM_INT_ATTRIBUTE)
```

Comment:
just to double-check: are we not using `forge` here because of the assertion query capabilities on the monitor side? like for boolean we can write something like `value in (true, false)`, but for other types there are no ways (direct or indirect) to assert the type?

Reply:
yes, you can only say `value:*` which means a wildcard. I would rather have my test cases based on a specific value than on a `wildcard`

Reply:
For numeric value, IIRC you can use comparisons (e.g.: `value:>10`). With that if we define a specific range we can ensure that the values are correct in the monitor